### DISCUSSION TITLE
Thảo luận thuật ngữ "tail-heavy"

### DISCUSSION THREAD

**Discussion by thanhcsf** [mlbvn/handson-ml2-vn#20] [Terminology] [answered]
Phần 11 của chương E2E ML Project anh em **[đang đề cập](https://github.com/mlbvn/handson-ml2-book/pull/77/files#r571397600)** từ này, anh em thảo luận thêm ở đây nhé.

Mention @mlbvn/handson-ml2

**Comment by quangnhat185** [comment]
Dịch là `lệch đuôi` thì sao nhỉ?

**Reply by lkhphuc** [reply]
`lệch` có từ `skewed distriubtion` rồi @quangnhat185 .

**Comment by lkhphuc** [comment]
> câu này không dịch là nặng. Tail-heavy là dạng đuôi giảm mạnh, từ đuôi nặng nghe có vẻ không hợp.
Cần đưa vô Glossary để thảo luận.

Mình thấy dịch `heavy tail / fat tail` là `phân phối nặng đuôi` đúng rồi mà nhỉ. 
Nó chỉ các distribution mà có pdf càng xa mean càng giảm **chậm hơn** normal distribution chứ đâu phải giảm nhanh hơn.
![](https://static.seekingalpha.com/uploads/2017/7/19/27369723-15004896402394993_origin.png)

**Comment by ngcthuong** [comment]
Từ này thuộc dạng khá cơ bản chắc đã dùng trong giảng dạy nhiều rồi.
Không biết mình có từ dịch tham khảo của các sách căn bản khác không nhỉ?

**Comment by rootonchair** [comment, marked as answer]
Do em chưa tìm thấy giáo trình nào nhắc đến tính chất này và từ `đuôi nặng` được dùng trong một vài bài viết thống kê nên em nghĩ mình có thể chấp nhận cách sử dụng này

**Reply by thanhcsf** [reply]
Mình tạm chốt dịch theo hướng này nhé.
